src/components/Tool/Appeal/List/AppealTour/AppealTour.test.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { ThemeProvider } from '@mui/material/styles';
+import { render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import TestRouter from '__tests__/util/TestRouter';
+import { GqlMockedProvider } from '__tests__/util/graphqlMocking';
+import { AppealsWrapper } from 'pages/accountLists/[accountListId]/tools/appeals/AppealsWrapper';
+import { ListHeaderCheckBoxState } from 'src/components/Shared/Header/ListHeader';
+import { useMassSelection } from 'src/hooks/useMassSelection';
+import theme from 'src/theme';
+import { AppealTour } from './AppealTour';
+
+const accountListId = 'accountListId';
+const deselectAll = jest.fn();
+const toggleSelectAll = jest.fn();
+
+jest.mock('src/hooks/useMassSelection');
+
+(useMassSelection as jest.Mock).mockReturnValue({
+  selectionType: ListHeaderCheckBoxState.Unchecked,
+  isRowChecked: jest.fn(),
+  toggleSelectAll,
+  deselectAll,
+  toggleSelectionById: jest.fn(),
+});
+
+const mockEnqueue = jest.fn();
+jest.mock('notistack', () => ({
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  ...jest.requireActual('notistack'),
+  useSnackbar: () => {
+    return {
+      enqueueSnackbar: mockEnqueue,
+    };
+  },
+}));
+
+const Components = ({ showTour = true }) => {
+  const appealId = showTour ? ['123', 'list', 'tour'] : ['123', 'list'];
+  return (
+    <ThemeProvider theme={theme}>
+      <TestRouter
+        router={{
+          query: { accountListId, appealId },
+          pathname:
+            '/accountLists/[accountListId]/tools/appeals/[[...appealId]]',
+          isReady: true,
+        }}
+      >
+        <GqlMockedProvider>
+          <AppealsWrapper>
+            <AppealTour />
+          </AppealsWrapper>
+        </GqlMockedProvider>
+      </TestRouter>
+    </ThemeProvider>
+  );
+};
+
+describe('AppealTour', () => {
+  beforeEach(() => {
+    deselectAll.mockClear();
+    toggleSelectAll.mockClear();
+  });
```

Comment:
These manual mock resets shouldn't be necessary. jest seems to do that automatically for each test.